firebase-installations/src/main/java/com/google/firebase/installations/CrossProcessLock.java

```diff
@@ -51,9 +51,10 @@ static CrossProcessLock acquire(Context appContext, String lockName) {
       // This method blocks until it can retrieve the lock.
       lock = channel.lock();
       return new CrossProcessLock(channel, lock);
-    } catch (IOException e) {
+    } catch (IOException | Error e) {
       // Certain conditions can cause file locking to fail, such as out of disk or bad permissions.
       // In any case, the acquire will fail and return null instead of a held lock.
+      // NOTE: In Java 7 & 8, FileKey creation failure wraps IOException into Error.
```

Comment:
It does not always wrap I think - we only got 1 developer mentioning this to us.
I think we should say "`might wrap`" instead of "`wraps`".
How about adding a link to https://bugs.openjdk.java.net/browse/JDK-8025619?

Reply:
Done.